packages/state-transition/src/cache/types.ts

```diff
@@ -8,6 +8,7 @@ export type BeaconStateAltair = CompositeViewDU<SSZTypesFor<ForkName.altair, "Be
 export type BeaconStateBellatrix = CompositeViewDU<SSZTypesFor<ForkName.bellatrix, "BeaconState">>;
 export type BeaconStateCapella = CompositeViewDU<SSZTypesFor<ForkName.capella, "BeaconState">>;
 export type BeaconStateDeneb = CompositeViewDU<SSZTypesFor<ForkName.deneb, "BeaconState">>;
+export type BeaconStateElectra = CompositeViewDU<SSZTypesFor<ForkName.electra, "BeaconState">>;
```

Comment:
maybe make a TODO to clean up these types (like we've done with our basic forky ssz types)

This would involve having something like:
```ts
export type BeaconStateViewDU<F extends ForkName = ForkName> = CompositeViewDU<SSZTypesFor<F, "BeaconState">>
```

as well as
```ts
export type CachedBeaconState<F extends ForkName = ForkName> = BeaconStateViewDU<F> & BeaconStateCache;
```